### ISSUE TITLE
some issue about dataloader wflw

### ISSUE THREAD

**Issue by lucaskyle** [HRNet/HRNet-Facial-Landmark-Detection#15]
wflw.py  the data loader of wflw.

in line 63:
img = np.array(Image.open(image_path).convert('RGB'), dtype=np.float32)

after that, normalizing the img:
img = (img/255.0 - self.mean) / self.std

why dont u just use 
PIL and pytorch.transform to normalize while training and testing?

**Comment by wondervictor** [comment]
Hi, you're right. We would adopt it.

**Comment by lucaskyle** [comment]
thx,  i got it
